public/wp-content/themes/ColoredCow/page-allevents.php

```diff
@@ -35,15 +39,16 @@
                     <div class="all-events-venue"><i class='fa fa-map-marker fa-lg' aria-hidden='true'></i>&nbsp;<?php echo $venue ?></div>
                     <br>
                 </div>
-                <div class="col-sm-12 col-md-4 col-lg-4 request-button">
-                    <a role="button" class="btn btn-warning btn-lg btn-block" data-toggle="modal" data-target="#request-modal" data-whatever="@mdo" data-id="<?php echo $id ?>">Request Invite
+                <div class="col-sm-12 col-md-4 col-lg-4 event-button">
+                    <a role="button" class="btn btn-warning btn-lg btn-block request-button" data-toggle="modal" data-target="#request-modal" data-whatever="@mdo" data-id="<?php echo $id ?>">Request Invite
                     </a>
                 </div>
             </div>
             <hr>
-            <?php 
-            }
-        }
+        <?php 
+                    }
+                }
+            } 
```

Comment:
use endwhile, endif, endfor. These braces don't should which block is ending. Better for code reading.